app/poros/csv_cafe_importer.rb

```diff
@@ -0,0 +1,32 @@
+class CSVCafeImporter
+  class << self
+    def import_file(file)
+      CSV.foreach(
+        file, 
+        headers: true,
+        encoding:'iso-8859-1',
+        header_converters: lambda { |header| header.downcase.tr(' ', '_') }) do |row|
+          data = row.to_h
+          data['name'] = data.delete('cafã©/restaurant_name')
+          data['notes'] = data.delete(data['nil'])
+          StreetCafe.create(data)
+      end
+      check_duplicates(file)
+    end
+
+    private
+
+    def check_duplicates(file)
+      if (CSV.read(file).length - 1) == StreetCafe.all.length
```

Comment:
This method is here in such that if you imported a csv the first time it wouldn't check for duplicate records. Any time after that this method will check for duplicate records and then remove them from the database.